controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -352,6 +359,12 @@ func (r *StorageClassRequestReconciler) reconcileRadosNamespace() error {
 		if blockPoolName == "" {
 			blockPoolName = fmt.Sprintf("%s-cephblockpool", r.storageCluster.Name)
 		}
+		if r.StorageClassRequest.Annotations[controllers.StorageRequestForceDeleteAnnotation] == "true" {
+			if r.cephRadosNamespace.Annotations == nil {
+				r.cephRadosNamespace.Annotations = make(map[string]string)
+			}
+			r.cephRadosNamespace.Annotations[RookCephResourceForceDeleteAnnotation] = "true"
+		}
```

Comment:
There is an add annotation util that does all of this. No need to write this logic yourself.